common/src/main/java/tachyon/security/authorization/FileSystemPermission.java

```diff
@@ -67,6 +67,14 @@ public FileSystemAction getUserAction() {
   }
 
   /**
+   * @param permission the digital representation of a {@link FileSystemPermission}
+   * @return the user {@link FileSystemAction}
+   */
+  public static FileSystemAction getUserAction(short permission) {
```

Comment:
On a different note, I think it is confusing that getUserAction() is a getter, while getUserAction(permission) is a factory.

I would rename the factories you have added in this PR from get* to create*.